etc/config/mojo.defaults.properties

```diff
@@ -0,0 +1,7 @@
+mojo.compilers=mojo
```

Comment:
If you want to get this on the live site, we'll need a `mojo.amazon.properties` with the configuration as you see fit. I think we can merge this as-is and then a follow up PR to add that though :)

Reply:
I'll try to quickly add the amazon properties. But we don't need to block this PR.  If I make it, great. If not, I can create a new PR.

Reply:
I'll hang on then ...

Reply:
I pushed a change with mojo-25.3.0, and mojo-nightly. We can work on getting nightly stood up.